packages/babel-preset-gatsby/src/index.js

```diff
@@ -0,0 +1,60 @@
+const r = m => require.resolve(m)
+
+function preset(context, options = {}) {
+  let { targets = null } = options
+
+  const stage = process.env.GATSBY_BUILD_STAGE || `test`
+
+  if (!targets) {
+    if (stage === `build-html`) {
+      targets = {
+        node: `current`,
+      }
+    } else {
+      targets = {
+        browsers: [`>0.25%`, `not dead`],
```

Comment:
I've hardcoded this now. We need to figure out how we can retrieve that from `./.cache/babelState.json` (see my comment on the `babel-loader-helpers.js` file).

Reply:
I wonder if it's possible to leave this up to users by adding a `.browserslistrc` file with these defaults to Gatsby starters. That way PostCSS can pick it up as well, which will result in a clean support table for the project.

Reply:
Still looking into that. Seems like the file is generated here:

https://github.com/gatsbyjs/gatsby/blob/f7d3231678a82fd1bf097e2c09d1f5dff4364496/packages/gatsby/src/internal-plugins/load-babel-config/gatsby-node.js#L8-L28

I understand this as being placed inside the `.cache` directory inside the **site root**, correct?

In this case I can probably just use the same pattern to access this file like we currently do:

https://github.com/gatsbyjs/gatsby/blob/f7d3231678a82fd1bf097e2c09d1f5dff4364496/packages/gatsby/src/utils/babel-loader-helpers.js#L4-L13

Which is what I'm doing right now.